build.sh

```diff
@@ -679,19 +693,21 @@ if [ $WITH_DEPS -gt 0 ]; then
     # abseil (compiled separately to apply a patch)
     #------------------------------------------------------------------------------
     abslPath=$sourcePrefix/abseil-cpp
-    if ! is_dependency_downloaded "absl_v${abslVersion}"; then
+    dep_absl=("absl_v${abslVersion}" "v1")
```

Comment:
What about putting all the "v1" in a central place like the version numbers? But it's not really important to change it. Just thinking out lout =)

Reply:
I also thought about that, but I don't think its that benefitial. Since the changes related to a dependency are probably done, where this (virtual) version is defined. My hope is to reduce the probability to overlook this necessary change, when working on a dependency.